spark/dl/src/main/scala/com/intel/analytics/bigdl/tensor/SparseTensorBLAS.scala

```diff
@@ -120,12 +120,14 @@ object SparseTensorBLAS {
         beta: T,
         r: Tensor[T])(implicit ev: TensorNumeric[T]): Unit = {
     (alpha, mat, vec, beta, r)  match {
-      case (alpha: Double, a: SparseTensor[Double], x: DenseTensor[Double],
-      beta: Double, y: DenseTensor[Double]) =>
-        dcoomv(alpha, a, x, beta, y)
-      case (alpha: Float, a: SparseTensor[Float], x: DenseTensor[Float],
-      beta: Float, y: DenseTensor[Float]) =>
-        scoomv(alpha, a, x, beta, y)
+      case (alpha: Double, a: SparseTensor[_], x: DenseTensor[_],
```

Comment:
@qiuxin2012 please take a look at this change. The SparseTensor[Float] will erase the float information

Reply:
Looks good